src/routes.js

```diff
@@ -17,11 +17,11 @@ import { saveToken } from './actions/auth'
 import { loadChallengeDetails } from './actions/challenges'
 import { connect } from 'react-redux'
 import { checkAllowedRoles, checkOnlyReadOnlyRoles, checkReadOnlyRoles } from './util/tc'
-import { setCookie, removeCookie, isBetaMode } from './util/cookie'
```

Comment:
was this the only place the cookies code was used? 🤔

Reply:
yes in this repository